### PR TITLE
Release 1.2.4

### PR DESCRIPTION
**Pull request by cetanu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- Enforce utf-8 for reading the readme in setup.py [#67](https://github.com/danielgtaylor/python-betterproto/pull/67)
+- Add intentionally failing test-cases for unimplemented bug-fixes [#55](https://github.com/danielgtaylor/python-betterproto/pull/55)
+- Only import types from grpclib when type checking [#52](https://github.com/danielgtaylor/python-betterproto/pull/52)
+- Re-organized test cases [#51](https://github.com/danielgtaylor/python-betterproto/pull/51)
+- Improve performance of serialize/deserialize by caching type information of fields in class [#46](https://github.com/danielgtaylor/python-betterproto/pull/46)
+- Adding basic support for client streaming [#45](https://github.com/danielgtaylor/python-betterproto/pull/45)
```
Comment (nat-n) [inline CHANGELOG.md:15]: Not yet on master, I'm still working on it and will update or replace the PR soon.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- Enforce utf-8 for reading the readme in setup.py [#67](https://github.com/danielgtaylor/python-betterproto/pull/67)
+- Add intentionally failing test-cases for unimplemented bug-fixes [#55](https://github.com/danielgtaylor/python-betterproto/pull/55)
+- Only import types from grpclib when type checking [#52](https://github.com/danielgtaylor/python-betterproto/pull/52)
+- Re-organized test cases [#51](https://github.com/danielgtaylor/python-betterproto/pull/51)
```
Comment (nat-n) [inline CHANGELOG.md:13]: Not sure about the value of putting non-functional updates in the change log.

Comment (boukeversteegh) [reply CHANGELOG.md:13]: Agreed :-) Refactorings and small internal improvements we could leave out

Comment (cetanu) [reply CHANGELOG.md:13]: Sounds good to me, feel free to bundle or exclude any that aren't particularly relevant to users

**Comment by nat-n** [comment]
This should be squashed down to a single commit on merge.

**Pull request merged**